doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -553,64 +560,6 @@ with corresponding nitrogen fluxes:
- +----------------------------------+----------------------------+
- | BDT boreal                       |             0.00           |
- +----------------------------------+----------------------------+
- | BES temperate                    |             0.00           |
- +----------------------------------+----------------------------+
- | BDS temperate                    |             0.00           |
- +----------------------------------+----------------------------+
- | BDS boreal                       |             0.00           |
- +----------------------------------+----------------------------+
- | C\ :sub:`3` arctic grass         |             0.00           |
- +----------------------------------+----------------------------+
- | C\ :sub:`3` grass                |             0.00           |
- +----------------------------------+----------------------------+
- | C\ :sub:`4` grass                |             0.00           |
- +----------------------------------+----------------------------+
- | Temperate Corn                   |             0.00           |
- +----------------------------------+----------------------------+
- | Spring Wheat                     |             0.00           |
- +----------------------------------+----------------------------+
- | Temperate Soybean                |             0.00           |
- +----------------------------------+----------------------------+
- | Cotton                           |             0.00           |
- +----------------------------------+----------------------------+
- | Rice                             |             0.00           |
- +----------------------------------+----------------------------+
- | Sugarcane                        |             0.00           |
- +----------------------------------+----------------------------+
- | Tropical Corn                    |             0.00           |
- +----------------------------------+----------------------------+
- | Tropical Soybean                 |             0.00           |
- +----------------------------------+----------------------------+
- | Miscanthus                       |             0.70           |
- +----------------------------------+----------------------------+
- | Switchgrass                      |             0.70           |
- +----------------------------------+----------------------------+
-
-Whereas food C and N was formerly transferred to the litter pool, CLM5 routes food C and N to a grain product pool where the C and N decay to the atmosphere over one year, similar in structure to the wood product pools. Biofuel and removed-residue C and N is also routed to the grain product pool and decays to the atmosphere over one year. Additionally, CLM5 accounts for the C and N required for crop seeding by removing the seed C and N from the grain product pool during harvest. The crop seed pool is then used to seed crops in the subsequent year.
```

Comment:
Shouldn't this stay as well?

Reply:
Above paragraph that is...

Reply:
A lot of info in that paragraph was redundant with text already included above. I added text above to cover the rest.